### ISSUE TITLE
feature request...

### ISSUE THREAD

**Issue by ronmurphy** [picotorrent/picotorrent#268]
Tried it, works well, I have a few suggestion ... like pure leech mode.. it would delete the torrent only once the sharing / seeding ratio hits 0.0 (qttorrent has this mode.. kind of.)

it needs anonymous mode, where it can require encryption before downloading...

But it is VERY memory friendly and most definitely not gaudy at all, it is simple, easy and just works, thanks for the app!

**Comment by vktr** [comment]
Thanks for the requests! I'll probably split this into two issues later on.

**Comment by vktr** [comment]
I've added the option to force encryption, it will be released with v0.11. If pure leech mode is something necessary, please open another issue and describe in detail what it entails.
